tests/simulation/test_simulator.py

```diff
@@ -17,6 +69,9 @@ def test_one_node(self):
         gen_tx1.start()
         self.simulator.run(60 * 60)
 
+        # FIXME: the setup above produces 0 new blocks and transactions
+        # self.assertGreater(manager1.tx_storage.get_count_tx_blocks(), 3)
```

Comment:
Should we create an issue to handle it later?

Reply:
Definitely, just did it: #397